modules/readme/Makefile

```diff
@@ -3,7 +3,7 @@ export README_FILE ?= README.md
 export README_YAML ?= README.yaml
 export README_TEMPLATE_FILE ?= $(BUILD_HARNESS_PATH)/templates/README.md
 export README_TEMPLATE_YAML := $(if $(findstring http,$(README_YAML)),$(README_YAML),$(BUILD_HARNESS_PATH)/templates/$(README_YAML))
-export INCLUDES ?= $(file://$(shell pwd)/?type=text/plain)
+export FILES ?= $(file://$(shell pwd)/?type=text/plain)
```

Comment:
No, rename to `README_INCLUDES` - the objective is to "namespace" environment variables related to the readme generation so they don't conflict. Generic names like `INCLUDES` or `FILES` are likely to "overload" other envs.